### DISCUSSION TITLE
sbb站

### DISCUSSION THREAD

**Discussion by hdjsjjdgrhj** [share121/inter-knot#1021] [其他]
![Screenshot_2024-08-10-21-10-16-336_tv danmaku bili](https://github.com/user-attachments/assets/a3da8a43-a783-4b9a-8f5f-73c26eca555b)

把我号封了是什么意思

**Comment by Signedinn** [comment]
是不是你的发布了什么不该发布的东西

**Comment by xiubachi** [comment]
是不是呗当成片哥了

**Comment by Y-0930** [comment]
太惨了，是不是账号被片哥盗用来

**Comment by DonQuixote084** [comment]
365天啊？之前干过什么，之前有被七天的记录吗？
